src/index.html

```diff
@@ -7,8 +7,86 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="logo moyo">
+      </a>
+      <nav class="navi">
+        <ul class="navigation">
+          <li>
+            <a
+            href="#"
+            class="navi-link is-active"
+            >
+            Apple
+            </a>
```

Comment:
Need to fix everywhere
```suggestion
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
```